webpack/webpack-workspace-resolve-plugin/README.md

```diff
@@ -0,0 +1,35 @@
+# @rushstack/webpack-workspace-resolve-plugin
```

Comment:
Add this plugin to the webpack configs in `heft-web-rig`, and in other webpack projects in this repo?

Reply:
This plugin is not usable by itself. I first need to build the tool that generates the cache file before it can be used. I wanted to merge this in so that I could use it to test generated cache files, though.